app/controllers/api/api_controller.rb

```diff
@@ -46,6 +55,14 @@ def auth_identity
       }
     end
 
+    def branding_info
+      {
+        :custom_brand      => Settings.server.custom_brand,
+        :custom_logo       => Settings.server.custom_logo,
+        :custom_login_logo => Settings.server.custom_login_logo
```

Comment:
What are these settings and where do they come from?  The only one in the settings template is the `custom_logo`, and that one is just a boolean.

Reply:
@Fryguy https://github.com/ManageIQ/manageiq/pull/17773 forgot to include, sorry